libbpfgo/helpers/kernel_features.go

```diff
@@ -0,0 +1,111 @@
+package helpers
+
+import (
+	"bufio"
+	"bytes"
+	"compress/gzip"
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+
+	"golang.org/x/sys/unix"
+)
+
+type KernelConfig map[string]string
+
+// NewKernelConfig simply creates the underlying map used by
+// the KernelConfig, you must use InitKernelConfig to read
+// in the values of the config
+func NewKernelConfig() KernelConfig {
+	kconfig := make(map[string]string)
+	return kconfig
```

Comment:
could we inline this?

```suggestion
	return make(map[string]string)
```

Reply:
Yup, good call. Interestingly the compiler inlines it anyway!